docs/guide/device-and-platform-support.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Device & Platform Support"
+type: guide
+layout: docs
+parent_section: guide
+order: 5
+---
+
+Devices and platform support depends on how well the browsers support certain
+devices and APIs. A-Frame supports both flat (3D on a normal screen) and WebVR
+experiences.  Though its focus is heavily VR. We will break up support into two
+categories: flat and VR.
+
+<!--toc-->
+
+## Support for Flat Experiences
+
+Support for flat experiences primarily depends on the browsers' WebGL support.
+We can see what browsers support WebGL at [Can I use WebGL?][caniusewebgl].
+
+## Support for VR Experiences
+
+Support for VR experiences, along with WebGL support, depends on the browsers'
+WebVR API support. The browsers that support the WebVR API *at the moment you
+read this* depends on the browsers and not on A-Frame. As more and more
+browsers adopt the WebVR APIs, they will inherit support from A-Frame.
+
+You can see what browsers *currently* support the WebVR API at [Is WebVR
+Ready?][iswebvrready].
+
+This version of A-Frame supports these forms of the WebVR API:
+
+| WebVR API                     | Details                                                                             |
+|-------------------------------|-------------------------------------------------------------------------------------|
+| [WebVR 1.0][webvr-1.0]        | Primary support.                                                                    |
+| Deprecated `getVRDevices` API | Supported through the [WebVR Polyfill][webvr-polyfill].                             |
+| WebVR Polyfill                | Supports mobile devices using device motion and orientation APIs (Cardboard-style). |
+
+### Support for Controllers
+
+*Out-of-the-box*, A-Frame supports `6DoF` controllers using Brandon Jones'
```

Comment:
link to https://en.wikipedia.org/wiki/Six_degrees_of_freedom - or at least mention "(e.g., HTC Vive, Oculus Touch, PSVR controllers)"